### PR TITLE
Add Access to MIRO

### PR DESCRIPTION
**Pull request by ChristopherHilton**
### What is this PR trying to achieve?
Adding the Access property to MIRO digital-only image JSON sample: there will need to be an indication in the JSON that this image is cleared to be viewed.  Appropriate explanation will be added to the markdown file
### Who is this change for?
Platform team then digital experience

### CODE REVIEW

**Comment by ChristopherHilton** [comment]
Corresponding Markdown file changes in #663

**Comment by alexwlchan** [comment]
Build is failing with the following errors:

```
/data/MIRO_sample_image_of_digital-only_image.json: line 45, col 9, Expected '}' and instead saw 'identifiers'.
/data/MIRO_sample_image_of_digital-only_image.json: line 45, col 22, Expected ']' and instead saw ':'.
/data/MIRO_sample_image_of_digital-only_image.json: line 46, col 11, Expected '}' and instead saw '['.
/data/MIRO_sample_image_of_digital-only_image.json: line 46, col 11, Unrecoverable syntax error. (83% scanned).
```

File: ontologies/MIRO_sample_image_of_digital-only_image.json [hunk 1]
```diff
@@ -41,6 +41,7 @@
         "id": "no819h6",
         "type": "item",
         "licence": "CC-BY-NC",
+        "access": "Open"
```
Comment (alexwlchan) [inline ontologies/MIRO_sample_image_of_digital-only_image.json]: you're missing a comma on the end of this line

**Comment by ChristopherHilton** [comment]
Necessary changes now made to ensure JSON is valid, and code passes checks, so merging this

**Pull request merged**